### PR TITLE
Created mandatory argument 'validated task name' when running MongoDB migrations...

### PR DESCRIPTION
**Pull request by vchske**
- Created mandatory argument 'validated task name' when running MongoDB migrations to force prior validations.
- Added mongo to mongo online as a scenario handled by the extension
- Minor cleanup
- Bumped version to 0.7.1

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

**Comment by williexu** [comment]
@temandr can you review as codeowner?

File: src/dms-preview/setup.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = "0.7.0"
+VERSION = "0.7.1"
```
Comment (temandr) [inline src/dms-preview/setup.py]: Change version to 0.8.0.

**Comment by williexu** [comment]
@vchske can you merge with the master branch?

**Comment by vchske** [comment]
@williexu  Merged with latest and looks good for merging into the master branch.

**Pull request merged**